### DISCUSSION TITLE
Trying to use TileMap with LDTK Plugin but I can't get the map to appear!

### DISCUSSION THREAD

**Discussion by icarito** [excaliburjs/Excalibur#3318] [Q&A] [answered]
I'm not sure what I'm missing from the examples!
I think I'm loading all the resources, and there's no error at sight, but also, not map!
I wondered if it was an instance of https://discord.com/channels/1195771303215513671/1317152887469772810 but I think not because I tried with 0.29.3.
Please guidance @eonarheim ! 
I have a running example at https://icarito.github.io/roke/ 
and the code can be found at: https://github.com/icarito/roke/blob/tilemap/src/main.jsx
Thanks in advance!

**Comment by eonarheim** [comment, marked as answer]
I tweaked the vite config, you in theory don't need the ldtkPlugin (the TSX version was becasue react typescript file extension collides with the Tiled Tileset extension, both have .tsx)

```typescript
export default defineConfig({
  plugins: [preact()],
  base: "/roke/",
  optimizeDeps: {
    exclude: ['excalibur']
  },
  build: {
    assetsInlineLimit: 0, // excalibur cannot handle inlined xml in prod mode
    sourcemap: true,
    rollupOptions: {
      output: "umd",
    },
  },
});
```

Then I ran a bake layers on the IntGrid auto tile layer (I think the LDtk plugin doesn't know how to handle these yet)

<img width="745" alt="image" src="https://github.com/user-attachments/assets/e171f094-aa71-4aee-9ffe-07070979d584" />

That makes it work

![image](https://github.com/user-attachments/assets/360e8348-05e8-4589-8991-6f49b4841755)


I'll see if I can add the int grid auto tile layer support without too much effort

**Reply by icarito** [reply]
Thank you so much for your help! If I'm able to help with this, I'd love to.
At least I hope my testing is helping and I can provide useful feedback.

**Comment by eonarheim** [comment]
For future explorers this has been fixed in the latest versions of excalibur and the tilemap plugins that now support esm across the board
